_members/maizy-brasher.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Maizy Brasher
+image: images/team/Maizy_Brasher.JPG
+description: intermediate information science professional/professional research assistant
+role: intermediate information science professional/professional research assistant
```

Comment:
I think this needs to be one of the set roles here: _data/roles.yaml. The description may need to be changed to PRA also.